projects/fal/src/faldbt/project.py

```diff
@@ -512,7 +512,6 @@ def __init__(
 
         telemetry.log_api(
             action="faldbt_initialized",
-            additional_props={"config_hash": self._config.hashed_name()},
```

Comment:
Why are we deleting this?

Reply:
This is being set inside `log_api` already, results in no functionality change

https://github.com/fal-ai/fal/blob/3e955628aed3ec728b2d644b84727d3677918f4e/projects/fal/src/fal/telemetry/telemetry.py#L297-L298